src/crypto/CHIPCryptoPAL.h

```diff
@@ -74,17 +74,10 @@ constexpr size_t kEmitDerIntegerWithoutTagOverhead = 1; // 1 sign stuffer
 constexpr size_t kEmitDerIntegerOverhead           = 3; // Tag + Length byte + 1 sign stuffer
 
 /*
- * Worst case is OpenSSL, so let's use its worst case and let static assert tell us if
- * we are wrong, since `typedef SHA_LONG unsigned int` is default.
- *   SHA_LONG h[8];
- *   SHA_LONG Nl, Nh;
- *   SHA_LONG data[SHA_LBLOCK]; // SHA_LBLOCK is 16 for SHA256
- *   unsigned int num, md_len;
- *
- * We also have to account for possibly some custom extensions on some targets,
- * especially for mbedTLS, so an extra sizeof(uint64_t) is added to account.
+ * Worst case is TI mbedtls hardware accelerated context, so let's use its
```

Comment:
could we keep the previous notes as 'considered sizes' ?

The fact that platform-specific items appear here is a bit concerning: if we need to allocate space for this, why would a non-ti platform pay the price for the TI HW accelerated size? Also with the previous code, why would mbedTLS pay the cost for the openssl max size.

@pan-apple @emargolis for feedback here: can we have some way to have more exact info here? This seems like an api failure of sorts.